R/Simulations-class.R

```diff
@@ -196,6 +196,7 @@ Simulations <- function(fit,
 #' @slot rho_est (`numeric`)\cr vector of final posterior median rho estimates
 #' @slot sigma2w_est (`numeric`)\cr vector of final posterior median sigma2W estimates
 #' @slot fit_biomarker (`list`)\cr with the final dose-biomarker curve fits
+#' @slot stop_report matrix of stopping rule outcomes
```

Comment:
please don't put this here because it is already document in the parent class `Simulations`